samples/TagHelperSample.Web/Areas/Admin/Views/Admin/Index.cshtml

```diff
@@ -0,0 +1,7 @@
+﻿@*
+    For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
+*@
+@{
+}
+
+<a asp-action="Index" asp-controller="Home" asp-area="">Go home!</a>
```

Comment:
Nope. The way Razor handles bound properties of type `string` prevents the more correct use of `null` to reference the root. That is, `asp-area=""` is unfortunately the same as `asp-area="@null"` -- except the second performs worse. And using an `object` property would be unhelpful for all the other cases.